tests/poll.rs

```diff
@@ -201,6 +204,49 @@ fn registry_behind_arc() {
     handle2.join().unwrap();
 }
 
+#[test]
+fn poll_no_timeout_not_affected_by_deregister() {
```

Comment:
Can you move this to the `regressions.rs` file, then the comment below won't be needed.

Reply:
done